lib/fp.js

```diff
@@ -71,31 +66,29 @@ const zip = (
   return res;
 };
 
-const replicate = (
-  // Create array of replicated values
-  count, // number, new array length
-  elem // value to replicate
-  // Returns: array, replicated
-) => (Array.from({ length: count }, () => elem));
-
-const zipWith = (
-  // Zip arrays using specific function
-  fn, // function, for zipping elements with index i
-  ...arrays // array of arrays
-  // Returns: array
-  // Element with index i of resulting array is result
-  // of fn called with arguments from arrays
-) => zip(...arrays).map(args => fn(...args));
-
-const curryUntil = (
-  // Curry function until the condition is met
-  condition, // function, (argsI, argsParts) returns boolean
-  // argsI is arguments for i-th currying
-  // argsParts is array of args given for currying from first to i-th currying
-  fn, // function, which will be curried
-  ...args // array, arguments for fn
-  // Returns: function, curried
-) => {
+// Create array of replicated values
+//   count - <number>, new array length
+//   elem - <any>, value to replicate
+// Returns: <array>, replicated
+const replicate = (count, elem) => (Array.from({ length: count }, () => elem));
+
+// Zip arrays using specific function
+//   fn - <Function>, for zipping elements with index i
+//   arrays - <Array[]>, arrays to be zipped
+// Returns: <Array>, zipped, element with index i of resulting array is result
+//          of fn called with arguments from arrays
```

Comment:
Why is this indented? You haven't indented similar comments in other places.

Reply:
I think we should indent such places everywhere at least to be consistent.